keyboards/ymdk/oni_hhkb/keymaps/default/keymap.c

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2025 chrisgve <https://github.com/chrisgve>
```

Comment:
Simple maps are better done in `keymap.json` files. Not required as far as I know, but preferred.

Reply:
Again, what should I do here? Is it required or good to have? If required, I'm happy to comply. If not, can we keep it the way it is?

Reply:
Fine the way it is.